debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -490,6 +504,9 @@ protected void handleInsert(Event event) throws InterruptedException {
             logger.debug("Skipping previously processed row event: {}", event);
             return;
         }
+        if (ignoreDmlEventByGtidSource) {
+            return;
```

Comment:
See https://gitter.im/debezium/dev?at=58949a5de836bf7010c8566d, but in this case use `io.debezium.connector.mysql` for the context (which are names of Java packages or classes). I'll add a document to the website that goes into more specifics.